agent/iskakalunan.py

```diff
@@ -66,7 +66,7 @@ def run(self):
             subprocess.call(['python', 'runtest.py', '-h', '>', 'result'])
             return
 
-        subprocess.call(['git', 'checkout', 'master'])
+        subprocess.call(['git', 'checkout', 'dev'])
```

Comment:
I think we should probably have some configuration file so that it can be more flexible on choosing the branch to be checked out?